framework/behaviors/AttributeBehavior.php

```diff
@@ -80,6 +80,12 @@ class AttributeBehavior extends Behavior
      */
     public $value;
     /**
+     * @var array an array of the values indexed by attribute name that will be assigned to the current attributes respectively by attribute name.
+     * Will be assigned if $this->value is NULL
+     * The single value same at @see $value
```

Comment:
I'd clarify description a bit

key - attribute name
value - corresponding value
Example:
\`\`\`php
'values' => [
    'number' => [$this, 'getNextNumber'],
    'status' => 'updated'
]
\`\`\`
@see value for possible values